### DISCUSSION TITLE
Get altitude telemetry to edgetx?

### DISCUSSION THREAD

**Discussion by soostdijck** [iNavFlight/inav#9130] [Q&A]
Hello,

I'm trying to get altitude telemetry to my edgeTX radio (v2.8.3). I'm using a Speedybee F405 WING stack with iNav 6.1.1 firmware, with a mateksys ELRS-R24-S receiver connected via CRSF. 

The barometer is working on the stack, and I don't have a GPS attached since I don't need it. I'm getting 18 sensors on my edgtx (TX16S). Including "VSpd" for a vario (so the barometer is definitely doing something).

However I don't get a altitude telemetry, is it possible to get this to the radio? 

Also, for some reason [Smartport seems to be doing a lot more telemetry](https://github.com/iNavFlight/inav/blob/master/docs/Telemetry.md#available-smartport-sport-sensors) is CRSF still being worked on?

Thanks!

**Comment by dz0ny** [comment]
You need to select "Discover sensors" under models Telemetry settings. You can then have custom property under telemetry.
